source/features/resolve-comments.js

```diff
@@ -0,0 +1,74 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+import {getUsername} from '../libs/utils';
+import * as icons from '../libs/icons';
+
+const RESOLVED_TEXT_REGEX = /^Comment resolved by @([a-zA-Z0-9-]+)/;
+
+export default () => {
+  for (const comment of select.all('.comment.timeline-comment')) {
+    const body = select('.comment-body', comment);
+    const isResolved = RESOLVED_TEXT_REGEX.test(body.innerText);
```

Comment:
`.innerText` => `.textContent`

Reply:
Needs your thoughts in https://github.com/sindresorhus/eslint-plugin-unicorn/issues/156